posts/2018-12-20-Rust-1.31.1.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Announcing Rust 1.31.1"
+author: The Rust Release Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.31.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.31.1 is as easy as:
+
+```
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.31.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1311-2018-12-20
+
+## What's in 1.31.1 stable
+
+This patch release fixes a build failure on on `powerpc-unknown-netbsd` by
+way of [an update to the `libc`
+crate](https://github.com/rust-lang/rust/pull/56562).
```

Comment:
```suggestion
crate](https://github.com/rust-lang/rust/pull/56562) used by the compiler.
```

I feel like otherwise it might communicate slightly the wrong message (i.e., libc crate is somehow special to the compiler/distribution).